pom.xml

```diff
@@ -256,6 +262,12 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.opentest4j</groupId>
+            <artifactId>opentest4j</artifactId>
```

Comment:
What has changed that we need to add it?

Reply:
I _think_ we started to use some JUnit 5 transitive dependency in the `hazelcast` test jar:

https://github.com/hazelcast/hazelcast-mono/blob/ccfb4f85d798b58f86852b1d6035bc0129c35893/hazelcast/hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java#L38

and 

https://github.com/hazelcast/hazelcast-mono/blob/ccfb4f85d798b58f86852b1d6035bc0129c35893/hazelcast/hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java#L391-L393